docs/building-for-tv.md

```diff
@@ -4,190 +4,8 @@ title: Building For TV Devices
-
-```tsx
-const TVEventHandler = require('TVEventHandler');
-
-class Game2048 extends React.Component {
-  _tvEventHandler: any;
-
-  _enableTVEventHandler() {
-    this._tvEventHandler = new TVEventHandler();
-    this._tvEventHandler.enable(this, function (cmp, evt) {
-      if (evt && evt.eventType === 'right') {
-        cmp.setState({board: cmp.state.board.move(2)});
-      } else if (evt && evt.eventType === 'up') {
-        cmp.setState({board: cmp.state.board.move(1)});
-      } else if (evt && evt.eventType === 'left') {
-        cmp.setState({board: cmp.state.board.move(0)});
-      } else if (evt && evt.eventType === 'down') {
-        cmp.setState({board: cmp.state.board.move(3)});
-      } else if (evt && evt.eventType === 'playPause') {
-        cmp.restartGame();
-      }
-    });
-  }
-
-  _disableTVEventHandler() {
-    if (this._tvEventHandler) {
-      this._tvEventHandler.disable();
-      delete this._tvEventHandler;
-    }
-  }
-
-  componentDidMount() {
-    this._enableTVEventHandler();
-  }
-
-  componentWillUnmount() {
-    this._disableTVEventHandler();
-  }
-}
-```
-
-- _Dev Menu support_: On the simulator, cmd-D will bring up the Dev Menu, similar to iOS. To bring it up on a real Apple TV device, make a long press on the play/pause button on the remote. (Please do not shake the Apple TV device, that will not work :) )
-
-- _TV remote animations_: `RCTTVView` native code implements Apple-recommended parallax animations to help guide the eye as the user navigates through views. The animations can be disabled or adjusted with new optional view properties.
-
-- _Back navigation with the TV remote menu button_: The `BackHandler` component, originally written to support the Android back button, now also supports back navigation on the Apple TV using the menu button on the TV remote.
-
-- _TabBarIOS behavior_: The `TabBarIOS` component wraps the native `UITabBar` API, which works differently on Apple TV. To avoid jittery re-rendering of the tab bar in tvOS (see [this issue](https://github.com/facebook/react-native/issues/15081)), the selected tab bar item can only be set from JavaScript on initial render, and is controlled after that by the user through native code.
-
-- _Known issues_:
-
-  - [ListView scrolling](https://github.com/facebook/react-native/issues/12793). The issue can be worked around by setting `removeClippedSubviews` to false in ListView and similar components. For more discussion of this issue, see [this PR](https://github.com/facebook/react-native/pull/12944).
-
-</TabItem>
-</Tabs>
+**To get full React Native for Tv documentation, visit [react-native-tvOS Wiki](https://github.com/react-native-tvos/react-native-tvos/wiki)**
```

Comment:
```suggestion
```

Reply:
This Wiki is not fully up to date and does not provide full documentation, so we should omit this change.